src/main/java/DialogBox.java

```diff
@@ -0,0 +1,59 @@
+import java.io.IOException;
+import java.util.Collections;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
```

Comment:
I think these should be coupled in groups of 2.
Like
import *****
import *****

import *****
import *****

Just check once